setup.py

```diff
@@ -51,7 +51,7 @@ def str_from_file(name):
     # transitive dependencies:
     #   urllib3: MIT
     #   aiohttp: Apache 2.0
-    "elasticsearch[async]==7.9.1",
+    "elasticsearch[async]==7.10.1",
```

Comment:
I was under the impression the point-in-time APIs I'm using were added in elasticsearch-py 7.10.  if that ends up not being the case, I'll revert this, otherwise I'll open the (blocking) PR and discuss timing internally with the team

Reply:
Sorry for not being clear enough. Yes, we need the 7.10 client for this PR. I was proposing that we raise a separate PR just to upgrade the client which we will merge prior to this PR. In the case something goes wrong the cause is easier to isolate. Does it make more sense now?

Reply:
Yep!